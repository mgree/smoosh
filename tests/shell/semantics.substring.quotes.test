FOO="a?b"
[ "${FOO#*"?"}" = b ] && echo OK1
FOO="abc"
[ "${FOO#"${FOO%???}"}" = "$FOO" ] && echo OK2
