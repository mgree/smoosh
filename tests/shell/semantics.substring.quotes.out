OK1
OK2
